homeassistant/components/aemet/strings.json

```diff
@@ -0,0 +1,32 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "[%key:common::config_flow::abort::already_configured_location%]"
+        },
+        "error": {
+            "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "api_key": "[%key:common::config_flow::data::api_key%]",
+                    "latitude": "[%key:common::config_flow::data::latitude%]",
+                    "longitude": "[%key:common::config_flow::data::longitude%]",
+                    "mode": "[%key:common::config_flow::data::mode%]",
+                    "name": "Name of the integration"
+                },
+                "description": "Set up AEMET OpenData integration. To generate API key go to https://opendata.aemet.es/centrodedescargas/altaUsuario",
+                "title": "AEMET OpenData"
+            }
+        }
+    },
+    "options": {
```

Comment:
This whole section too, is no longer used?  I assume you would need to clean up the translations provided in the PR, but I'm not really sure on this point.